components/image-builder-api/typescript/src/sugar.ts

```diff
@@ -235,6 +238,12 @@ export class PromisifiedImageBuilderClient {
                     }
                 }
 
+                log.info("Needs image build?", {
+                    status: resp.getStatus(),
+                    source: request.getSource(),
+                    forceRebuild: request.getForceRebuild(),
+                    triggeredBy: request.getTriggeredBy(),
```

Comment:
What values does this field contain? Is it potentially sensitive?

Reply:
> Is it potentially sensitive?

It is not